.github/workflows/publish-release.yml

```diff
@@ -14,14 +14,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          # This is to guarantee that the most recent tag is fetched.
```

Comment:
Nit: Probably worth mentioning here that we need this for `update-major-version-tag.sh`